Userland/Utilities/pkg/AvailablePort.cpp

```diff
@@ -42,7 +36,7 @@ static Optional<AvailablePort&> find_port_package(HashMap<String, AvailablePort>
     return port->value;
 }
 
-ErrorOr<int> AvailablePort::query_details_for_package(HashMap<String, AvailablePort>& available_ports, HashMap<String, InstalledPort>& installed_ports, StringView package_name, bool verbose)
+ErrorOr<int> AvailablePort::query_details_for_package(HashMap<String, AvailablePort>& available_ports, HashMap<String, InstalledPort> const& installed_ports, StringView package_name, bool verbose)
```

Comment:
Why does this return ErrorOr<int> ? It always returns 0. There's not even any TRY.

Reply:
Honestly, I had same thought, but I kept it as it is. I will make it `int`. But since return 0 is always returned, void seems better here.